src/provider/systemdmanager/service.rs

```diff
@@ -0,0 +1,321 @@
+use std::collections::HashMap;
+
+use kubelet::container::Container;
+use kubelet::pod::Pod;
+use phf::{phf_map, phf_ordered_set};
```

Comment:
Not sure I fully understood your meaning, I have now brought Map and OrderedSet into the scope and used phf itself so that for the macro calls I specify phf:: ...  was that what you meant?